maro/simulator/scenarios/oncall_routing/carrier.py

```diff
@@ -21,7 +21,8 @@ class Carrier(NodeBase):
     delayed_order_num = NodeAttribute("i")
     total_delayed_time = NodeAttribute("i")
 
-    in_stop = NodeAttribute("i")    # In a stop or on the way to next stop
+    in_stop = NodeAttribute("i")  # In a stop or on the way to next stop
+    next_departure_tick = NodeAttribute("i")  # Tick that the carrier leaves the current stop
```

Comment:
I think it should not be an attribute of the carrier. It should be some information like "actual duration" that is unknown until it happens. So if you need it, maybe maintain it in the internal dict (in BE) is better.